packages/terra-clinical-onset-picker/src/OnsetPicker.module.scss

```diff
@@ -8,6 +8,7 @@
 
     .field-inline {
       display: inline-block;
+      vertical-align: top;
```

Comment:
After a bit of investigation -- @jeremyfuksa AND @emilyrohrbough were [right on the nose](https://github.com/cerner/terra-clinical/pull/596#discussion_r364337689). All we needed to do was fix `vertical-align` on the field component. 

I couldn't figure out why it wasn't already coming through directly to the Onset-picker, since Onset-picker uses the Terra-Field component and this is already fixed for IE, but then I found that Onset-picker adds it's own className for `field-inline` to each form field, and does not use the `isInline` prop as I was expecting (probably to avoid needing to override the extra spacing, might be my guess). Example here:
https://github.com/cerner/terra-clinical/blob/44bc65409b07fe940c9d0bb5add6bd45d96ce4d5/packages/terra-clinical-onset-picker/src/OnsetPicker.jsx#L373-L376

For a short-term & quick fix, I have added the `vertical-align: top` to the `.field-inline` class here in onset picker css to match the same styling that Terra-Field has in it's `.field-inline` class here: 
[../Field.module.scss#L9-L13](https://github.com/cerner/terra-core/blob/bc075f286cd9a548094a79f2f721ec526e2b251a/packages/terra-form-field/src/Field.module.scss#L9-L13), and removes the need for any additional media queries, etc.

However, it might be worthwhile to revisit and test swapping out the internal class to the actual isInline prop with any appropriate spacing overrides as needed, with some extensive cross-browser testing to make sure it doesn't introduce any other layout bugs. Can be discussed later.